packagingbuild/xenial/Dockerfile

```
@@ -38,22 +38,24 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
 #
 
 
-# install python development
-RUN apt-get update && \
-    apt-get -y install build-essential python-dev python python-virtualenv
+# Install python 3.6 & development
+# from the PPA as it's not available in base distro
+RUN sudo add-apt-repository ppa:deadsnakes/ppa && \
```

You should already be running as `root`, so you shouldn't need `sudo` here, and if you don't pass in `-y` or `--yes` then the `add-apt-repository` will hang waiting for user input.
```suggestion
RUN add-apt-repository --yes ppa:deadsnakes/ppa && \
```

---

Right, sudo somehow went through copy-paste while testing on a local VM.
Good catch :+1: